requirements-optional.txt

```diff
@@ -15,6 +15,10 @@ boto3
 azure-storage-blob
 google-cloud-storage
 
+# for testing
```

Comment:
Can we put that in `requirements-test.txt` or something like that?

Reply:
@wkuopt Thanks for working on this. One question: I see you can now pass folder name in `path` variable in the dictionary for aws. If somebody wants to deploy on root directory of bucket (as it is right now), can we leave path and it will deploy to root? (current behavior).

I see you have changed the docstrings in persistence.py but it's not exposed to end-users. Unfortunately you have to change docstring part in individual files `classification.py` `regression.py` `clustering.py` `anomaly.py`.

Thanks

Reply:
Hey, thanks for the comments!

@Yard1 I have created a separate file for the test library.
@pycaret I have modified those files, and indicated that path is optional. And yes, if it's not provided, then it will be deployed to the root directory of that bucket :)